main/src/main/scala/sbt/Defaults.scala

```diff
@@ -164,7 +164,10 @@ object Defaults extends BuildCommon {
   private[sbt] lazy val globalJvmCore: Seq[Setting[_]] =
     Seq(
       compilerCache := state.value get Keys.stateCompilerCache getOrElse CompilerCache.fresh,
-      classLoaderLayeringStrategy :== ClassLoaderLayeringStrategy.AllLibraryJars,
+      classLoaderLayeringStrategy := {
+        if (turbo.value) ClassLoaderLayeringStrategy.AllLibraryJars
```

Comment:
Maybe this should be (ThisBuild / turbo).value? I was experimenting with this and noticed that if I ran  `ThisBuild / turbo := true` at the command line that it had no effect.

Reply:
Actually, after more experimentation, I think maybe the best option is to move this into coreDefaultSettings. That allows the user to set turbo at the Global, ThisBuild and project level and have it work as expected.

Reply:
Doh.

Reply:
> That allows the user to set turbo at the Global, ThisBuild and project level and have it work as expected.

This makes sense.

Reply:
6878fb6cdbbdf038b97120d7242f782698df6763